### DISCUSSION TITLE
Proposal: Payment Gateway Integration for Pakistan

### DISCUSSION THREAD

**Discussion by TALHA0713** [medusajs/medusa#10734] [Feature Requests]
Hi Medusa Team,

I’m Muhammad Talha, and I’d like to propose adding a custom payment gateway integration for Pakistan (e.g., JazzCash, Easypaisa). I couldn’t find an existing plugin for this.
Could you guide me on how to create and integrate such a plugin? I’ve reviewed the documentation but need help understanding the workflow and steps.
Looking forward to your guidance and suggestions!
Best regards,
Muhammad Talha

**Comment by Taimoorkhan1122** [comment]
would definitely love to see official support for these payment providers.
